intranet/apps/eighth/models.py

```diff
@@ -1806,3 +1807,25 @@ def update_weighted(self):
     def __str__(self):
         act_set = self.activity_set.all()
         return "{} and {}: {}".format(act_set.first(), act_set.last(), self.count)
+
+
+class EighthMetricsCollector:
+    def describe(self):
+        yield GaugeMetricFamily("intranet_eighth_next_block_signups", "The number of signups for the next block")
+        yield GaugeMetricFamily("intranet_eighth_next_block_signups_remaining", "The number of users who are not signed up for the next block")
```

Comment:
I'm debating whether this should be `signups_remaining` or `remaining_signups`.

Reply:
I prefer `signups_remaining` because `remaining` is a qualifier on `signups`, just like `next_block` is a qualifier on `eighth`. Ultimately, though, it doesn't matter too much.